client/src/Features/PublishAssignment/PublishSuccessMessageBar.tsx

```diff
@@ -50,7 +50,11 @@ const PublishSuccessMessageBarInner = ({
 
   if (isShown) {
     return (
-      <MessageBar messageBarType={MessageBarType.success} isMultiline={false} styles={themedClassNames(styles)}>
+      <MessageBar 
+        messageBarType={MessageBarType.success} 
+        isMultiline={false} 
+        styles={themedClassNames(styles)}
+      >
```

Comment:
Any specific reason to make this change?

Reply:
No just a bit of formatting. I can retract the same. Although I ran prettier again and it did some more changes to the import statements as well. I will commit the same. Please let me know in case it isn't needed as well!